docs/src/modules/java/pages/value-entity.adoc

```diff
@@ -23,7 +23,7 @@ The following `counter_domain.proto` file defines our "Counter" Value Entity. Th
 [source,proto]
 .src/main/proto/value-entities/counter_domain.proto
 ----
-include::example$valueentity-counter/src/main/proto/value-entities/counter_domain.proto[]
+include::example$java-valueentity-counter/src/main/proto/value-entities/counter_domain.proto[]
```

Comment:
I prefix the `valueentity-counter` with "java" in this PR. I'm planning to prefix all samples in another PR to accomodate for the upcoming scala sdk. But since I touched that sample here, it was better to prefix it now to avoid conflict.